### ISSUE TITLE
Installer includes adware

### ISSUE THREAD

**Issue by blahpers** [Pulover/PuloversMacroCreator#265]
Including adware in your installer that installs without consent is a violation of GitHub's Acceptable Use Agreement.  Remove it immediately.

**Comment by wedoprods** [comment]
I've never experienced this issue whenever I click "Decline" it normally respects my choice and doesn't install it. I see its interesting that some people have this issue and some doesn't.

**Comment by EmuAl** [comment]
yes i've just had this webcompanion program installed on my pc now i can't get it off, not good and severly not happy, there was no decline option. sort it out

**Comment by wedoprods** [comment]
I have never seen it happen where there was never a decline option.

**Comment by TeaBarry** [comment]
There was no decline option. Bad faith behaviour.

**Comment by Pulover** [comment]
It doesn't show again if you had already installed/uninstalled the offer.

Em seg., 10 de jun. de 2024, 08:47, TeaBarry ***@***.***>
escreveu:

**Comment by ToneLeMoan** [comment]
Declined adware (already regretting even downloading software with this option). After install and run program hung, needing to be force closed but using 10% of CPU. Dodgy as ****.

**Comment by mark-wiemer** [comment]
Hey I'm just curious, where is the adware? I just installed this and didn't notice anything weird. What's it called and where does it install?

**Comment by mark-wiemer** [comment]
The adware is "Web Companion" and it can be found and uninstalled through the standard Windows menu. Ref https://www.reddit.com/r/AutoHotkey/comments/yy5ovn/warning_pullovers_macro_recorder_has_malware.

And here's the [GitHub Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies)

**Comment by mark-wiemer** [comment]
FYI, I'm working on fixing this in https://github.com/mark-wiemer/adware-free-macro-creator. No promises!

**Comment by mark-wiemer** [comment]
Fixed: https://github.com/mark-wiemer/adware-free-macro-creator/releases/tag/v5.4.2
